pyrasterframes/src/main/python/docs/nodata-handling.pymd

```diff
@@ -132,29 +135,137 @@ masked.select(rf_no_data_cells('blue_masked'), rf_tile_sum('mask')).show(10)
 
 It's also nice to view a sample.
 
-```python show_masked
+```python
 sample = masked.orderBy(-rf_no_data_cells('blue_masked')).select(rf_tile('blue_masked'), rf_tile('scl')).first()
 display(sample[0])
 ```
 
 And the original SCL data.
 
-```python show_scl
+```python
 display(sample[1])
 ```
 
-## NoData in Arithmetic Operations
+## Changing a Tile's NoData Values
+
+One way to mask a tile is to make a new tile with a user defined NoData value. We will explore this method below. First, lets create a rasterframe from a tile with values of 0, 1, 2, and 3.
+
+```python create_dummy_tile
+tile_size = 100
+x = np.zeros((tile_size, tile_size), dtype='int16')
+
+for i in range(4):
+    x[:, i*tile_size//4:(i+1)*tile_size//4] = i
```

Comment:
I have no idea what this is doing. @vpipkt Are we assuming that users are adept at NumPy? I'd prefer to have a comment that tells us the shape and content of what's being constructed here.

Reply:
Maybe this is a little too fancy, but I mainly just wanted the user to ignore it since it and take the result for granted. I had hoped that the description above the cell block and the picture rendered below would show what that code section does. I could change the tile to have random values of 0, 1, 2, and 3 instead. Less pretty, but slightly simpler numpy.

Reply:
@emc5ud I think a comment describing what you're creating is sufficient.

Reply:
Decided to add an explanation